templates/support/livepatch.html

```diff
@@ -0,0 +1,160 @@
+        </p>
+        <cite class="p-pull-quote__citation">Masaaki Hirose, IT Platform Department, DeNA</cite>
+      </blockquote>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="u-equal-height">
+      <div class="col-7 suffix-1">
+        <h2>How to enable the Canonical Livepatch Service</h2>
+        <ol class="p-list-step">
+          <li class="p-list-step__item">
+            <h3 class="p-list-step__title">
+              <span class="p-list-step__bullet">1</span>
+              Generate your credentials
+            </h3>
+            <p>Via the <a href="https://auth.livepatch.canonical.com/">Canonical Livepatch portal</a>.</p>
+          </li>
+          <li class="p-list-step__item">
+            <h3 class="p-list-step__title">
+              <span class="p-list-step__bullet">2</span>
+              Install the livepatch daemon
+            </h3>
+            <div class="p-code-snippet">
+              <input class="p-code-snippet__input" value="sudo snap install canonical-livepatch" readonly="readonly">
+              <button class="p-code-snippet__action">Copy to clipboard</button>
+            </div>
+          </li>
+          <li class="p-list-step__item">
+            <h3 class="p-list-step__title">
+              <span class="p-list-step__bullet">3</span>
+              Enable it on your system
+            </h3>
+            <div class="p-code-snippet">
+              <input class="p-code-snippet__input" value="sudo canonical-livepatch enable [TOKEN]" readonly="readonly">
+              <button class="p-code-snippet__action">Copy to clipboard</button>
+            </div>
+          </li>
+        </ol>
+      </div>
+      <div class="col-5">
+        <div class="p-card">
+          <div class="p-heading-icon--muted">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img p-heading-icon__img--small" src="{{ ASSET_SERVER_URL }}75902002-news-feed-strip-stock_ebook.svg" alt="" />
+              <h4 class="p-heading-icon__title">Datasheet</h4>
+            </div>
+          </div>
+          <h3>Learn more about Livepatch</h3>
+          <ul class="p-list">
+            <li class="p-list__item is-ticked">Detailed product overview</li>
+            <li class="p-list__item is-ticked">System requirements</li>
+            <li class="p-list__item is-ticked">Answers to frequently asked questions</li>
+          </ul>
+          <p>
+            <a class="p-button" href="{{ ASSET_SERVER_URL }}ac3aa269-DS_Canonical_Livepatch_Service_screen-AW_08.17.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'resource', 'eventAction' : 'Livepatch datasheet', 'eventLabel' : 'from ubuntu.com/server/livepatch', 'eventValue' : undefined });"><span class="p-link--external">Download the datasheet</span></a>
```

Comment:
The button and the p here should have `u-no-margin--bottom` to align the padding around the card.